f5_openstack_agent/lbaasv2/drivers/bigip/test/test_ssl_profile.py

```diff
@@ -0,0 +1,119 @@
+# coding=utf-8
+# Copyright 2014-2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from f5_openstack_agent.lbaasv2.drivers.bigip.ssl_profile import \
+    SSLProfileError
+from f5_openstack_agent.lbaasv2.drivers.bigip.ssl_profile import \
+    SSLProfileHelper
+
+
+import mock
+import pytest
+
+
+class TestSSLProfileHelper(object):
+    def test_get_ssl_profile_count(self):
+        bigip = mock.MagicMock()
+        bigip.tm.ltm.profile.client_ssls.get_collection =\
+            mock.MagicMock(return_value=[1, 2, 3, 4])
+        assert(SSLProfileHelper.get_client_ssl_profile_count(bigip) == 4)
+
+    def test_create_client_ssl_profile_exists(self):
+        bigip = mock.MagicMock()
+        bigip.tm.ltm.profile.client_ssls.client_ssl =\
+            mock.MagicMock(return_value=True)
+        rv = SSLProfileHelper.create_client_ssl_profile(
+            bigip, 'testprofile', 'testcert', 'testkey')
+        assert(rv is None)
```

Comment:
This is a fine test. We often try to add a little more substance by making assertions about some (or all) the calls made to the mocked object like that in https://github.com/F5Networks/f5-openstack-agent/blob/master/f5_openstack_agent/lbaasv2/drivers/bigip/ssl_profile.py#L38. Just a tip. No need to change anything.